client/my-sites/themes/current-theme/index.jsx

```diff
@@ -65,28 +67,47 @@ class CurrentTheme extends Component {
 				<QueryActiveTheme siteId={ siteId } />
 				{ currentThemeId && <QueryCanonicalTheme themeId={ currentThemeId } siteId={ siteId } /> }
 				<div className="current-theme__current">
-					{ showScreenshotPlaceholder && <div className="current-theme__img-placeholder" /> }
-					{ showScreenshot && (
-						<img src={ currentTheme.screenshot + '?w=150' } className="current-theme__img" alt="" />
-					) }
-					<span className="current-theme__label">{ translate( 'Current Theme' ) }</span>
-					<span className="current-theme__name">{ text }</span>
-				</div>
-				<div
-					className={ classNames( 'current-theme__actions', {
-						'two-buttons': Object.keys( options ).length === 2,
-					} ) }
-				>
-					{ map( options, ( option, name ) => (
-						<CurrentThemeButton
-							name={ name }
-							key={ name }
-							label={ option.label }
-							icon={ option.icon }
-							href={ currentThemeId && option.getUrl( currentThemeId ) }
-							onClick={ this.trackClick }
-						/>
-					) ) }
+					<div className="current-theme__details">
+						{ showScreenshotPlaceholder && <div className="current-theme__img-placeholder" /> }
+						{ showScreenshot && (
+							<img
+								src={ currentTheme.screenshot + '?w=150' }
+								className="current-theme__img"
+								alt=""
+							/>
+						) }
+						<div className="current-theme__description">
+							<div className="current-theme__title-wrapper">
+								<span className="current-theme__label">
+									{ currentTheme && currentTheme.name && translate( 'Current Theme' ) }
+								</span>
+								<span className="current-theme__name">{ text }</span>
+							</div>
+							<p>
+								{ translate( 'This is the active theme on your site.' ) }{ ' ' }
```

Comment:
I'm curious if you've seen this `{ ' ' }` pattern elsewhere? I can see why you're using it, I'm just wondering if that issue has been solved any other way in Calypso. 🤔

Reply:
I've seen this pattern in other places, and *prettier* will add these on its own sometimes:
![prettier-update](https://user-images.githubusercontent.com/937354/124617558-a4d8ae80-de3c-11eb-9cc4-1597113346d7.gif)

Reply:
Yeah, this is a pretty standard React way to ensure spaces between items when you're cleaning up JSX with code linebreaks. Because React doesn't interpret a JSX linebreak as whitespace.